bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala

```diff
@@ -58,6 +58,7 @@ object Bundle {
       val tokenizer = "tokenizer"
       val vector_assembler = "vector_assembler"
       val one_hot_encoder = "one_hot_encoder"
+      val one_hot_encoder_v23 = "one_hot_encoder_v23"
```

Comment:
maybe we should call it "one_hot_encoder_estimator" for now, to match what Spark is calling it in 2.3?

Reply:
I wonder whether we shouldn't keep the two models completely separate on MLeap side as well, allowing us not to have to modify the serialization (OneHotEncoderOp) on the existing one hot encoder. I'm just thinking that one hot encoding is very common and with the changes as they are now, people looking to upgrade to a newer version of MLeap in order to take advantage of bug fixes and new features available would need to retrain & re-serialize all their models. 

I know that we don't guarantee backwards compatibility yet, but maybe in this case, we don't really need to break the backwards compatibility? What do you think @clocklear @hollinwilkins?

Reply:
So, a bit of info to add some clarity.

The old 1HE is just called `OneHotEncoder`.  It is a stand-alone feature transformer.
The new 1HE has an estimator component called `OneHotEncoderEstimator`.  The output of the `fit()` method is a `OneHotEncoderModel`, which is the transformer that Mleap actually cares about.

I'm not strongly opinionated on the naming of the constant in `Bundle.scala`, other than I don't really think it should be named after the estimator.  After all, Mleap doesn't care about estimators or anything else that happens at train time.

Reply:
Because we have 1 transformer and 1 core model for both v23 and previous versions, it would be best if we did not need to introduce an additional op name.

Default v23 to register "one_hot_encoder" as the V23 op, users can customize to the old one hot encoder if needed.

For below v23, default to the old one hot encoder.

Reply:
A better long-term solution would be to support multiple ops per op name, but that is a bit out of scope for this PR.

Reply:
@clocklear Sorry, I misread the changes in the op for the existing 1HE, makes sense now!

Reply:
MLeap support for 2.3 is relatively new, so I think Hollin was suggesting for 2.3 to default to the new 1HE op (since the old one is deprecated and will be removed in 3.0 anyways), while keeping the old 1HE op for the previous versions.

Reply:
Ahh, thanks for the clarification @ancasarb!  That makes a lot of sense.

Reply:
So, if we don't register the old 1HE in 2.3, the parity tests no longer pass (because we didn't register an op for the transformer).  Is that ok?

I don't think we should do this until 1HE is actually removed (in 3.0).  It's deprecated now, but still fully supported, so I think we should still support it.  Again, I could be in the minority here.

Reply:
I think you can try to create a single OneHotEncoderParitySpec class and then define the sparkTransformer something like this

```
  override val sparkTransformer: Transformer =
    if (org.apache.spark.SPARK_VERSION.equals("2.3.0")) {
// new 1HE pipeline
} else {
// old 1HE pipeline
}
```

Reply:
PR updated.